src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,18 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. This view can be found by toggling the tab in the upper right corner of the homepage.
+
+![trends Homepage](trends-homepage.png)
+
+Trending transactions are calculated first by filtering out transactions with large fluctuations in [throughput](/product/performance/metrics/#throughput-total-tpm-tps), then determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than 0, while improvements are percentage changes less than 0.
```

Comment:
Do we need to update this because of any changes to the logic for throughput filtering? IIRC we talked about potentially relaxing the requirements for the ratio of before/after volume (and it may be interesting to surface these changes to the customer even if transaction speed doesn't change much), especially since we can now use a t-test to throw out stuff that's not interesting.

Reply:
In the short-term we won't be surfacing throughput changes on the trends view, and for now we'll have the throughput filtering so I don't think this needs an update (but I'll keep this comment unresolved as a reminder to update this if we do remove the throughput filtering)